wgsl/index.bs

```diff
@@ -6941,21 +6941,21 @@ To declare a variable for accessing a particular output built-in *Y* from an ent
   * Otherwise, make the result type to be a structure, where one of the fields is the new variable.
 * Apply a `builtin(`*Y*`)` attribute to the result variable.
 
-Both input and output built-in variables may also be declared as members of
+Both input and output built-in values may also be declared as members of
 structures that are either entry point function parameters (for inputs) or the
 return type of an entry point (for outputs).
-The type of the structure member must match the type specified for the built-in variable.
+The type of the structure member must match the type specified for the built-in value.
 
-The `builtin` attribute must not be applied to a variables in [=module scope=],
-or the local variables in the function scope.
+The `builtin` attribute must not be applied to a variable or constant declared in [=module scope=],
+or in [=function scope=].
```

Comment:
You could just remove this this since the attribute specifies it can only be applied to entry point parameters, return types or members of a structure.